TranslatorReasonerAPI.yaml

```diff
@@ -737,11 +743,51 @@ components:
           items:
             type: string
           example: [inhibits, activates]
+        attributes:
+          type: array
+          description: >-
+            Edge attributes provided by this TRAPI web service.
+          items:
+            $ref: '#/components/schemas/MetaAttribute'
       required:
         - subject
         - predicate
         - object
       additionalProperties: false
+    MetaAttribute:
+      type: object
+      properties:
+        attribute_type_id:
+          $ref: '#/components/schemas/CURIE'
+          description: >-
+            Type of an attribute provided by this TRAPI web service
+            (preferably the CURIE of a Biolink association slot)
+          example: biolink:p_value
+        attribute_source:
+          type: string
+          description: >-
+            Source of an attribute provided by this TRAPI web service.
+          example: infores:chembl
+        original_attribute_names:
+          type: array
+          items:
+            type: string
+          description: >-
+            Names of an the attribute as provided by the source.
+        costraint_use:
+          type: boolean
+          description: >-
+            Indicates whether this attribute can be used as a query
+            constraint.
+          example: true
+        constraint_name:
```

Comment:
I think the constraint name is supposed to be the same original_attribute_name, so this may be redundant.

Reply:
Not necessarily as the same attribute can have different original names from different sources as shown in an example below.